guake/globals.py

```diff
@@ -93,6 +93,11 @@ def is_run_from_git_workdir():
         r"^\s*File\s\"(.*)\",\sline\s([0-9]+)",
     ),
     ("Python pytest report", r"^\s.*\:\:[a-zA-Z0-9\_]+\s", r"^\s*(.*\:\:[a-zA-Z0-9\_]+)\s",),
+    (
+        "line starts by 'ERROR in Filename:line' pattern (GCC/make). File path should exists.",
```

Comment:
can you update the release note yaml please?

Reply:
@gsemet , I've updated the release notes. Please have a look.
Also, I'm taking part in this year's Hacktoberfest, would you mind adding label `hacktoberfest-accepted` to this PR once its approved?